components/mpas-ocean/src/shared/mpas_ocn_gm.F

```diff
@@ -723,13 +723,18 @@ subroutine ocn_GM_compute_Bolus_velocity(statePool, &
                RossbyRadius(iEdge) = c_mode1*sqrt( 1.0_RKIND / &
                   (fEdge(iEdge)*fEdge(iEdge) + 2.0_RKIND*betaEdge(iEdge)*c_mode1 + 1.0E-15_RKIND))
 
-               ! If the RossbyRadius is larger than two cell widths, then this
-               ! edge is sufficiently well-resolved, and turn off the
-               ! parameterization.
-               if (RossbyRadius(iEdge).ge.2.0_RKIND*dcEdge(iEdge) .or. c_mode1.lt.0.1_RKIND) then
+               dc_Lr = dcEdge(iEdge)/RossbyRadius(iEdge)
```

Comment:
Should we add a small number (1.0E-15_RKIND) to the denominator to avoid possible division by 0?

Reply:
Good thought but it looks like that shouldn't be necessary. c_mode1 >= 1e-5 so Lr > 0 always.